docs/content/en/docs-dev/user-guide/managing-application/configuration-drift-detection.md

```diff
@@ -60,6 +60,12 @@ You can change the checking interval as well as [configure the notification](../
 
 You can also ignore drift detection for specified fields in your application manifests. In other words, even if the selected fields have different values between live state and Git, the application status will not be set to `Out of Sync`.
 
+##### Input format
+```yaml
+driftDetection:
+  ignoreFields:
+    - apiVersion:kind:namespace:name#path
+```
```

Comment:
I think we don't need this, just simple text like below is enough

Reply:
Don't you also need apiVersion:kind:namespace:name#path?
I think it's hard to understand this format with just example, what do you think?

Reply:
I saw the next comment. I understood it. Thank you.